drivers/clk/adi/clk-ad9545.c

```
@@ -1749,6 +1786,42 @@ static int ad9545_plls_setup(struct ad9545_state *st)
 			if (ret < 0)
 				return ret;
 
+			tmp = pll->profiles[j].fast_acq_excess_bw;
+			for (k = 0; k < ARRAY_SIZE(ad9545_fast_acq_excess_bw_map); k++)
+				if (tmp == ad9545_fast_acq_excess_bw_map[k])
+					break;
+
+			if (k < ARRAY_SIZE(ad9545_fast_acq_excess_bw_map)) {
```

What if `k == ARRAY_SIZE(ad9545_fast_acq_excess_bw_map)`? This means that the dts property given by the user is wrong and we are silently ignoring it, right? Maybe this would make more sense?

```
if (k >= ARRAY_SIZE(ad9545_fast_acq_excess_bw_map)`) {
        dev_err(...);
        return -EINVAL;
}
```
If the user is using this and for some reason just got it wrong, we should be verbose about it...